### ISSUE TITLE
midi_rnd.ttl doesn't declare units prefix

### ISSUE THREAD

**Issue by mxmilkiib** [ycollet/psi-plugins#4]
Via Yruama in #lv2

**Comment by ycollet** [comment]
Thanks for the report. I will check soon.

**Comment by trebmuh** [comment]
pinging @yruamalairba

**Comment by ycollet** [comment]
I tried a fix.
It loads fine under qtractor.
On Fedora 31, the lv2 package was tool old and unit:midinote doesn't exist in the lv2 units extension

**Comment by YruamaLairba** [comment]
maybe some tool like [lv2lint](https://open-music-kontrollers.ch/lv2/lv2lint/) or [sord_validate](http://drobilla.net/software/sord) will help you for the future.

**Comment by ycollet** [comment]
Yes, certainly. sord is normally available with some drobilla package. 
I will check to add an atomatic check pass in the spec file
